api/spec/v1.yaml

```
@@ -2024,6 +2038,22 @@ components:
             The number of addresses that have a nonzero balance of this token,
             as calculated from Transfer events.
           example: 123
+    
+    AccountStats:
+      type: object
+      required: [num_txns]
+      properties:
+        total_sent:
+          description: The total number of tokens sent, in base units.
+          <<: *BigIntType
+        total_received:
+          description: The total number of tokens received, in base units.
+          <<: *BigIntType
+        num_txns:
+          description: The total number of transactions this account was involved with.
```

```suggestion
          description: The total number of transactions this account submitted.
```

I'd say that's more indicative of account activity?

---

Hm I'm ambivalent on this one. I agree that submitted txs are perhaps more interesting, but I looked at a couple other block explorers and while most don't have this field, the ones that do track transactions that an account was involved with, [example](https://www.blockchain.com/explorer/addresses/btc/bc1q6ufjmqnzz0ck2mvdx5t6yq5enguz86ur29wqhx7wn7dlg5j5cg0swy9wfl)

---

+1 to ambivalence. I'd think block explorers generally want a higher activity stat.